receiver/simpleprometheusreceiver/README.md

```diff
@@ -0,0 +1,81 @@
+# Simple Prometheus Receiver
+
+### Overview
+
+The `prometheus_simple` receiver is a wrapper around the [prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/prometheusreceiver).
+This receiver provides a simple configuration interface to configure the prometheus
+receiver to scrape metrics from a single target. Here's an example config.
+
+```yaml
+    receivers:
+      prometheus_simple:
+        collection_interval: 10s
+        use_service_account: true
+        endpoint: "172.17.0.5:9153"
+        tls_enabled: true
+        ca_file: "/path/to/ca"
+        cert_file: "/path/to/cert"
+        key_file: "/path/to/key"
+        insecure_skip_verify: true
```

Comment:
I think this is no longer correct. This needs to be under `tls_config`.

Reply:
The TLS options and the ones in `httpConfig` are embedded in the receiver config so these would still be at the top level.

Reply:
Oh, I didn't notice `tls_config` was removed. I was thinking we'd keep it but just add tls_enabled beside it.

Reply:
Ah yes, looking back at the example I gave above, I realized that's what we discussed too. Updated.